src/utilities/new_versions.jl

```diff
@@ -162,7 +164,8 @@ end
 
 function continue_with_pr(dep::DepInfo, bump_compat_containing_equality_specifier::Bool)
     # Determine if we need to make a new PR
-    if !isnothing(dep.version_spec) && dep.latest_version in dep.version_spec
+    if (!isnothing(dep.version_spec) && !isnothing(dep.latest_version)) &&
+       dep.latest_version in dep.version_spec
```

Comment:
looks like this change is required because `nothing in VersionSpec` doesn't have a method for it.